backend/model/top_container.rb

```diff
@@ -148,6 +148,11 @@ def self.sequel_to_jsonmodel(objs, opts = {})
       json['display_string'] = obj.display_string
       json['long_display_string'] = obj.long_display_string
 
+      obj.linked_archival_records.each do |archival_record|
+        json['linked_records'] ||= []
+        json['linked_records'] << ArchivalObject.to_jsonmodel(archival_record)
```

Comment:
As mentioned above, this could be a 'ref' to save calling to_jsonmodel on any linked record.  This could then be batch resolved only when required.